config/application.rb

```diff
@@ -26,7 +26,19 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
-    config.middleware.use(ActionDispatch::Cookies)
-    config.middleware.insert_after(ActionDispatch::Cookies, ActionDispatch::Session::CookieStore)
+    # config.middleware.use(ActionDispatch::Cookies)
+    # config.middleware.insert_after(ActionDispatch::Cookies, ActionDispatch::Session::CookieStore)
+
+    # in-mem session store. not very scalable, but can write new instance on redis or something
+    # using abstractstore
+    config.middleware.use(ActionDispatch::Session::MemCacheStore)
+
+    ## Cors
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*' # dev mode. not prod safe.
```

Comment:
Please use TODO (Your Name): task. If the task is a very important todo format it like this TODO URGENT (Your Name): task